### PR TITLE
Remove traces of restartRequired

### PR DESCRIPTION
**Pull request by Vimal-Raghubir**
As indicated in the commit messages, these portions of code were removed since restartRequired is not needed anymore. The issue that indicated this required fix is #3571.

### CODE REVIEW

File: frontend/src/app/containers/App/index.js [hunk 1]
```diff
@@ -63,7 +59,6 @@ import { localizationsSelector, negotiatedLanguagesSelector } from "../../select
 import { chooseTests } from "../../actions/varianttests";
 import addonActions from "../../actions/addon";
 import newsletterFormActions from "../../actions/newsletter-form";
-import RestartPage from "../RestartPage";
```
Comment (lmorchard) [inline frontend/src/app/containers/App/index.js:66]: Since the `RestartPage` container is no longer used, it should be removed from the project too

**Comment by fzzzy** [comment]
I'll take a look at this tomorrow.

**Comment by fzzzy** [comment]
@Vimal-Raghubir Thanks, this looks good, but there are conflicts now. Can you please pull from master into your branch, fix the conflicts, and push to your branch? You could also rebase the branch if you know how to do that and prefer that.

**Review by fzzzy — CHANGES_REQUESTED** [review]
Conflicts need resolving.

**Pull request merged**